Tests/SourceGenerators/SourceGeneratorAssemblyToProcess/Generated/Realm.SourceGenerator/Realms.SourceGenerator.RealmGenerator/CollectionErrors.diagnostics.cs

```diff
@@ -1,6 +1,6 @@
 ﻿[
   {
-    "Id": "REALM001",
+    "Id": "???",
```

Comment:
This file doesn't appear to be exercised in tests which is why I put ??? here - when we write tests for it, they should fail.